examples/lsp_spice.py

```diff
@@ -66,20 +68,54 @@
 shape_model = TrimeshShapeModel(V, F, N)
 
 # Load compressed form factor matrix from disk
-
 FF_path = 'lsp_compressed_form_factors.bin'
 FF = cff.CompressedFormFactorMatrix.from_file(FF_path)
+# print("Getting FF block") # what's this?
+# FF_gt = get_form_factor_block(shape_model)
```

Comment:
by the way, what's this (l. 78)? It's not really used in the original lsp_spice either (an old experiment maybe?)

Reply:
Yes, old code used to compare the compressed form factor matrix against the ground truth. This will be quite slow, so good to remove it.